projects/unrar/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "http://www.rarlab.com/"
+primary_contact: "roshal@rarlab.com"
+sanitizers:
```

Comment:
Do you want to be autocced to results as well. If yes, add yourself in auto_ccs

Reply:
Done